Sources/Core/Tracker/TrackerEvent.swift

```diff
@@ -39,8 +39,8 @@ class TrackerEvent : InspectableEvent, StateMachineEvent {
     
     private(set) var isService: Bool
     
-    init(event: Event, state: TrackerStateSnapshot? = nil) {
-        eventId = UUID()
+    init(event: Event, eventId: UUID, state: TrackerStateSnapshot? = nil) {
```

Comment:
This is coming too late, but setting the default value for the parameter would prevent some annoying changes in the tests:

```suggestion
    init(event: Event, eventId: UUID = UUID(), state: TrackerStateSnapshot? = nil) {
```